stempy/equalizer.cpp

```diff
@@ -0,0 +1,104 @@
+// contrast equalizer file
+#include "equalizer.h"
+
+// todo: parallel computation
+#ifdef VTKm
+#endif
+
+#include <cmath>
+#include <future>
+#include <algorithm>
+#include <iostream>
+
+
+namespace stempy 
+{
+    // hide helper functions
+    namespace 
+    {
+        // helper function that returns the binding result
+        std::vector<float> getcontainer(const STEMImage inImage, const int numBins)
```

Comment:
We are normally using `camelCase` for stempy, so `getContainer` would be a better name, I think.

Reply:
Sure, I will change it to getContainer, thank you.